.openshiftio/application.yaml

```diff
@@ -60,8 +72,8 @@ objects:
     strategy:
       sourceStrategy:
         from:
-          kind: ImageStreamTag
-          name: runtime-thorntail-health-check:latest
+          kind: DockerImage
+          name: 'registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift'
```

Comment:
Why?

Reply:
If we use imagestream here, so whenever there will be a change in base image, it will trigger a build. It will avoid that case.

Reply:
> If we use imagestream here, so whenever there will be a change in base image, it will trigger a build.

I'd say we _want_ that.

Reply:
In OSIO case we do not want to build in this case hence we have removed it.

Reply:
I don't know why OSIO doesn't want it, as this kind of automation is a big part of what makes OpenShift great, and I'm -0.9 to this change. I'm going to defer to higher authorities here. @kenfinnigan @balunasj ?